src/models/branches/lines.jl

```diff
@@ -9,12 +9,12 @@ struct Line <: Branch
     anglelimits::@NT(max::Float64, min::Float64) #Degrees
 
     function Line(name::String, available::Bool, connectionpoints::@NT(from::Bus, to::Bus), r::Float64, x::Float64, b::@NT(from::Float64, to::Float64), rate, anglelimits)
```

Comment:
we can remove the inner constructor completely.

Reply:
I have left it there because in most cases just one rate is passed and not a NT(from_to, to_from), so it checks if the rate is in correct format.

Reply:
That is a good idea, but then we should move this to an outer constructor.